thrift/thrift0.13/src/main/java/com/linecorp/armeria/server/thrift/ThriftServiceEntry.java

```diff
@@ -73,6 +76,18 @@ public Set<Class<?>> interfaces() {
         return metadata.interfaces();
     }
 
+    /**
+     * Returns the associated thrift method name in proto from Thrift function.
```

Comment:
I failed to get a reference from googling that `proto` and Thrift are used together. 
Where did you get the term? Otherwise, I prefer using `.thrift` file instead for this and the other usage. 
```suggestion
     * Returns the associated thrift method name in `.thrift` file from Thrift function.
```

Reply:
Yes, it's better